src/models/AppState.js

```diff
@@ -586,6 +586,15 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
         fn: function (policy) {
           return policy && policy.rememberDeviceByDefault;
         }
+      },
+      'factorsPolicyInfo' : {
```

Comment:
Nit: It seems like only `autoPushEnabled` is used from this `factorsPolicyInfo` object. If there was no intention to add anything more to this object, maybe just derive `autoPushEnabled` in the model as opposed to the full object?

Reply:
It might be used to pass in information of other factors in the future. Also autoPushEnabled is dependent on the factorID, and appState is loaded before okta-verify is, so I was not able to get the correct value of autoPushEnabled in time